x-pack/plugins/apm/public/components/alerting/utils/helper.ts

```diff
@@ -16,6 +16,13 @@ export interface AlertMetadata {
   end?: string;
 }
 
+export interface SearchConfiguration {
+  query: {
+    query: string | { [x: string]: any };
+    language: string;
+  };
+}
+
```

Comment:
This is already defined in `searchConfigurationSchema`. I don't think we need to define it again

Reply:
updated in bee1742c5c900538fe1131d9c0c0b6de350c6dcc